examples/bert/main.py

```diff
@@ -168,7 +177,7 @@ def forward(    self,
             loss = loss + local_target_loss
             
         # Compute NSP loss for network outputs. Only run this if we have passed network inputs.
-        if network is not None and next_sentence_labels is not None:
+        if query and next_sentence_labels is not None:
```

Comment:
Little nitpicky: can just keep indented and check for next sentence labels inside the `if query`